### ISSUE TITLE
Emissive pixels are incorrectly transparent(?) on heads

### ISSUE THREAD

**Issue by 0xFE0F** [exaskye/Ears#133]
Version: 1.20.1
Mod Loader: Fabric

Emissive pixels on player heads are not rendered _at all_
Screenshot of the bug in-action:

![2023-07-22_11 18 24](https://github.com/unascribed/Ears/assets/131938948/8262dd27-8c5e-4042-ab48-b027106e9947)

(Also sorry if this is bad, this is the first bug report I've _ever_ made)

**Comment by exaskye** [comment]
Yes, this is one of the reasons emissives are still beta. They work by drawing your entire player a second time without lighting using an alternate version of your skin that's just the emissive pixels. To avoid z-fighting, the places emissive pixels were on the vanilla skin are erased.

The way these are rendered has been pending a rework for a long time, likely via GL_POLYGON_OFFSET, but it's not something I've gotten around to.
